### PR TITLE
Add support for usernames

### PR DESCRIPTION
**Pull request by Salty-Coder**
Adds support for Discord's new username system.

Before             |  After
:-------------------------:|:-------------------------:
image  |  image

### CODE REVIEW

File: Plugins/RoleMembers/RoleMembers.plugin.js [hunk 1]
```diff
@@ -239,7 +239,8 @@ module.exports = !global.ZeresPluginLibrary ? Dummy : (([Plugin, Api]) => {
             const scroller = popout.querySelector(".role-members");
             for (const member of members) {
                 const user = UserStore.getUser(member.userId);
-                const elem = DOM.parseHTML(Utilities.formatString(itemHTML, {username: Utils.escapeHTML(user.username), discriminator: "#" + user.discriminator, avatar_url: ImageResolver.getUserAvatarURL(user)}));
+                const discriminator = user.discriminator != 0 ? "#" + user.discriminator : "";
+                const elem = DOM.parseHTML(Utilities.formatString(itemHTML, {username: Utils.escapeHTML(user.username), discriminator: discriminator, avatar_url: ImageResolver.getUserAvatarURL(user)}));
```
Comment (jaw0r3k) [inline Plugins/RoleMembers/RoleMembers.plugin.js:243]:
```suggestion
                const elem = DOM.parseHTML(Utilities.formatString(itemHTML, {username: Utils.escapeHTML(user.username), discriminator, avatar_url: ImageResolver.getUserAvatarURL(user)}));
```

**Comment by rauenzi** [comment]
This doesn't modify the source...

**Comment by Salty-Coder** [comment]
> This doesn't modify the source...

What do you mean by that?

**Comment by rauenzi** [comment]
This PR only modifies the distribution file and not the actual source of the plugin under `src`. That file should not be updated by PRs

**Comment by Salty-Coder** [comment]
> This PR only modifies the distribution file and not the actual source of the plugin under `src`. That file should not be updated by PRs

Ah. I didn't realise that.   
I'll make a new commit to the source file.

**Comment by rauenzi** [comment]
Please also remove the change to [Plugins/RoleMembers/RoleMembers.plugin.js](https://github.com/rauenzi/BetterDiscordAddons/pull/728/files/6fd6b9d0a2969d269910df5553acc8792fc8e3d4#diff-e3f7b17830c8b22c88fd37a195dd423432f88cfd7afb68ca36f9e821e25d23fb) otherwise it'll trigger an update on the BD site and there's more to add before a new release

**Comment by Salty-Coder** [comment]
> Please also remove the change to [Plugins/RoleMembers/RoleMembers.plugin.js](https://github.com/rauenzi/BetterDiscordAddons/pull/728/files/6fd6b9d0a2969d269910df5553acc8792fc8e3d4#diff-e3f7b17830c8b22c88fd37a195dd423432f88cfd7afb68ca36f9e821e25d23fb) otherwise it'll trigger an update on the BD site and there's more to add before a new release

Yep. I'm working on that.

**Comment by Salty-Coder** [comment]
Never removed commits before. Let me know if that's all ok or if I've messed something up.

**Comment by rauenzi** [comment]
For future reference, you don't have to actually remove any commits, you can just make a new commit that undoes your changes to that file, then when I merge I squash the commits so the number of commits made doesn't matter.

**Pull request merged**